arbnode/espresso/chain/metrics.go

```diff
@@ -0,0 +1,112 @@
+package chain
+
+import (
+	"context"
+	"sync"
+	"time"
+
+	espresso_client "github.com/EspressoSystems/espresso-network/sdks/go/client"
+	espresso_types "github.com/EspressoSystems/espresso-network/sdks/go/types"
+	espresso_common "github.com/EspressoSystems/espresso-network/sdks/go/types/common"
+	"github.com/offchainlabs/nitro/arbnode/espresso"
+)
+
+type EspressoChainClientMetrics struct {
```

Comment:
I think adding metrics is a good idea but it doesnt have to be a separate struct. We should think about if we can map the original espresso client in an interface and then if we want to enable metrics or not should be controlled by a boolean